messenger.rst

```diff
@@ -1810,6 +1810,17 @@ The transport has a number of options:
     If the queue name is suffixed by ``.fifo``, AWS will create a `FIFO queue`_.
     Use the stamp :class:`Symfony\\Component\\Messenger\\Bridge\\AmazonSqs\\Transport\\AmazonSqsFifoStamp`
     to define the ``Message group ID`` and the ``Message deduplication ID``.
+    You can also enable the
```

Comment:
1. This is only useful when using AWS
2. We should explain more what it does internally, when using it

Reply:
This bit has been added to the SQS part, so I guess your first point is resolved :slightly_smiling_face: 

I also reworded to be more precise on what this middleware does internally. I missed the two new interfaces. Thanks!